webapp/views/App/views/Analysis/Chain/store/actions/index.js

```diff
@@ -7,28 +7,13 @@ import { useAddNodeDefAnalysis } from './useAddNodeDefAnalysis'
 import { useOnSave } from './useOnSave'
 import { useOpenRStudio } from './useOpenRStudio'
 
-export const useActions = ({
-  attributesUuidsOtherChains,
-  setAtrributesUuidsOtherChains,
-  chain,
-  setChain,
-  dirty,
-  setDirty,
-  step,
-  setStep,
-  calculation,
-  setCalculation,
-}) => ({
-  onInit: useOnInit({ chain, setChain, step, setStep, calculation, setCalculation }),
-  onDismiss: useOnDismiss({ chain, setChain, dirty, setDirty }),
-  onSave: useOnSave({ chain, setChain, step, calculation }),
-  canSelectNodeDef: useCanSelectNodeDef({ chain }),
-  getAttributeUuidsOtherChains: useGetAttributeUuidsOtherChains({
-    attributesUuidsOtherChains,
-    setAtrributesUuidsOtherChains,
-    chain,
-  }),
+export const useActions = (dependencies) => ({
```

Comment:
better name for dependencies is props?
why not to list all the required props? ```{...}``` IDE provide more support with declared objects